### ISSUE TITLE
Reference guide link broken

### ISSUE THREAD

**Issue by jgarvin** [ZekeMedley/lust#31]
https://zmedley.com/lust

The link there to the reference guide gives me 404.

(Also does Lust work running in WASM?)

**Comment by 0xekez** [comment]
Does https://zmedley.com/lust/index.html work for you?

Should work with wasm fine but you'd need to change the code to make cranelift output that. :)

**Comment by jgarvin** [comment]
Yep, to be clear both links work, but only following the second link can you get to the reference page linked at the bottom. I got the first link from the README.md.
